src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -307,16 +307,10 @@ public static List<RelatedParty> getIndividualRelatedParties(Entity entity, Stri
         List<RelatedParty> relatedParties = new ArrayList<>();
         EntityRoleCode roleCode = getEntityRoleCode(entity);
         switch (roleCode) {
-            case APPOINTEE:
```

Comment:
Shouldn't there be failing tests due to these deletions?

Reply:
The only existing test was to check the REPRESENTATIVE returns an expected partyId and relationshipType (SOLICITOR). No coverage on the removed. I've added test for the new INTERPETER role.